core/state_transition.go

```diff
@@ -222,7 +222,10 @@ func (st *StateTransition) TransitionDb() (ret []byte, usedGas uint64, failed bo
 		}
 	}
 	st.refundGas()
-	st.state.AddBalance(st.evm.Coinbase, new(big.Int).Mul(new(big.Int).SetUint64(st.gasUsed()), st.gasPrice))
+	fee := new(big.Int).Mul(new(big.Int).SetUint64(st.gasUsed()), st.gasPrice)
+	if !st.evm.ChainConfig().IsDarvaza(st.evm.BlockNumber) {
+		st.state.AddBalance(st.evm.Coinbase, fee)
+	}
```

Comment:
Actually we can simplify this to just wrapping the original line now. `fee` was split out for something that didn't make the final version.
```suggestion
	if !st.evm.ChainConfig().IsDarvaza(st.evm.BlockNumber) {
		st.state.AddBalance(st.evm.Coinbase, new(big.Int).Mul(new(big.Int).SetUint64(st.gasUsed()), st.gasPrice))
	}
```